manifests/g/Google/GooglePlayGames/Beta/1.3.36.152/Google.GooglePlayGames.Beta.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Google.GooglePlayGames.Beta
+PackageVersion: 1.3.36.152
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://play.google.com/googleplaygames
+PackageName: Google Play Games Beta
+PackageUrl: https://play.google.com/googleplaygames
+License: proprietary
+Copyright: Copyright 2018 Google LLC
+ShortDescription: Take mobile gaming to your PC
+ManifestType: defaultLocale

```

```suggestion
Moniker: google-play-games
Tags:
- android
- google
- games
- gaming
- store
ManifestType: defaultLocale
```
Metadata from previous manifest.